src/macros.hpp

```diff
@@ -77,4 +77,20 @@ struct StaticNextPow2 {
   enum { value = StaticNextPow2Helper<8 * sizeof(size_t) - 1, N>::value };
 };
 
+#if !defined(CASS_STATIC)
+#  if (defined(WIN32) || defined(_WIN32))
+#    if defined(CASS_BUILDING)
+#      define CASS_IMPL_EXPORT __declspec(dllexport)
+#    else
+#      define CASS_IMPL_EXPORT __declspec(dllexport)
+#    endif
```

Comment:
This code copied from include/cassandra.h. In commit https://github.com/datastax/cpp-driver/commit/fb4c1a6f4c8824bc5198c7f404609e3a8d22a997 the message says that it solves some Windows warnings. But I can rewrite this code with single #define.